supply-chain-security/compromises/2016/gh-unicode.md

```diff
@@ -0,0 +1,11 @@
+# GitHub delivered password resets to wrong addresses
+
+GitHub's validation logic for password resets didn't account for a Unicode normalization feature that could have allowed impersonation for the purpose of account takeover.
+
+## Impact
+
+This was addressed by GitHub and neither GitHub or the original reporter mention any known attacks.
```

Comment:
Is there more information on this from the original reporter about potential impact?  perhaps changing this to unknown if we don't have any and cite GitHub and the original reporter not identifying known attacks.